platforms/macos/src/node.rs

```diff
@@ -523,6 +523,11 @@ declare_class!(
                 .unwrap_or(false)
         }
 
+        #[method(isAccessibilityEnabled)]
+        fn enabled(&self) -> bool {
+            self.resolve(|node| !node.is_disabled()).unwrap_or(true)
```

Comment:
If, for whatever reason, the node cannot be resolved, it should be marked as disabled. At least this is what Chromium does.

```suggestion
            self.resolve(|node| !node.is_disabled()).unwrap_or(false)
```